package.json

```diff
@@ -48,7 +48,8 @@
   "scripts": {
     "lint": "jshint examples lib test",
     "test": "mocha test/api/ --timeout 15000",
-    "pretest": "npm run lint"
+    "pretest": "npm run lint",
+    "payments": "mocha test/api/payments --timeout 15000"
```

Comment:
"payments" script is unnecessary. It will be picked up by "test": "mocha test/api/ --timeout 15000